nilearn/surface/surface.py

```diff
@@ -727,12 +746,14 @@ def load_surf_data(surf_data):
                 gii = _load_surf_files_gifti_gzip(surf_data)
                 data_part = _gifti_img_to_data(gii)
             else:
-                raise ValueError(('The input type is not recognized. %r was '
-                                  'given while valid inputs are a Numpy array '
-                                  'or one of the following file formats: .gii,'
-                                  ' .gii.gz, .mgz, .nii, .nii.gz, Freesurfer '
-                                  'specific files such as .area, .curv, .sulc,'
-                                  ' .thickness, .annot, .label') % surf_data)
+                raise ValueError(
+                    'The input type is not recognized. '
+                    f'{surf_data!r} was '
+                    'given while valid inputs are a Numpy array '
+                    'or one of the following file formats: .gii,'
+                    ' .gii.gz, .mgz, .nii, .nii.gz, '
```

Comment:
Take it or leave it: This list of extensions also appears 4 times so why not standardize is like the others

Reply:
yeah I was very tempted but felt like I was getting on a slippery slope of "but then I could also refactor this, and also this, and that, and this there..."

will do this one though. :-p

Reply:
Done. But of course I had do also hoist a couple of guard clauses because it was just too tempting.